README.md

```diff
@@ -1,8 +1,8 @@
-# BurgerNabijeBesluitendatabank (front-end)
+# Lokaal Beslist (front-end)
 
-[The front-end for BNB](https://burgernabije-besluitendatabank-dev.s.redhost.be/), a site that uses linked data to empower everyone in Flanders to consult the decisions made by their local authorities.
+[The front-end for Lokaal Beslist](https://lokaalbeslist-dev.s.redhost.be/), a site that uses linked data to empower everyone in Flanders to consult the decisions made by their local authorities.
```

Comment:
For the different frontends. The three urls are:
DEV: dev.lokaalbeslist.lblod.info
QA: lokaalbeslist.lblod.info
PROD lokaalbeslist.vlaanderen.be

I think in this case it's best to share the prod environment as 'The Frontend'

Reply:
In either case the url (https://lokaalbeslist-dev.s.redhost.be/) does not exist